addons/product/models/product_pricelist.py

```diff
@@ -131,8 +131,7 @@ def _compute_price_rule(self, products_qty_partner, date=False, uom_id=False):
         """
         self.ensure_one()
         if not date:
-            date = self._context.get('date') or fields.Date.today()
-        date = fields.Date.to_date(date)  # boundary conditions differ if we have a datetime
+            date = self._context.get('date') or fields.Datetime.now()
```

Comment:
This context key is absolutely degueulasse. But it's not your fault. ^^

Reply:
It's only one of the crappy things in the pricelist schmilblick currently :P.